catalog/security/catalog-security-filter/src/main/java/ddf/catalog/security/filter/plugin/FilterPlugin.java

```diff
@@ -100,16 +114,15 @@ public QueryResponse process(QueryResponse input)
             }
         }
 
-        logger.info("Filtered {} metacards, returned {}", filteredMetacards,
-                (newResults.size() - filteredMetacards));
+        logger.info("Filtered {} metacards, returned {}", filteredMetacards, newResults.size());
```

Comment:
Do we really need/want these two loggers here?
Also, interesting catch about the "size()- filteredMetacards"... wonder if someone was confusing size with capacity.